app/services/authentication/providers/osso.rb

```diff
@@ -0,0 +1,76 @@
+module Authentication
+  module Providers
+    # Osso authentication provider, uses omniauth-osso as backend.
+    # Osso is an open source service for adding SAML based SSO to
+    # your application. The Osso team added Osso as a provider
+    # in order to serve as a real-world example of how simple it
+    # is to integrate Osso.
+    #
+    # Learn more about Osso at https://ossoapp.com
+
+    class Osso < Provider
+      OFFICIAL_NAME = "SAML SSO".freeze
```

Comment:
I'd probably want to use ENV vars here if I were actually adding this to forem. if its an internal community, it might be "Sign in with EnterpriseCo Okta". but in multi-tenancy, we expect folks to be more generic and not use "Osso".